tailwind.config.js

```diff
@@ -17,6 +17,10 @@ module.exports = {
         'app-background': '#fff',
         orange: '#ff5a42',
       },
+      fontFamily: {
+        // prettier-ignore
+        paragraph: ['Raleway','sans-serif'],
```

Comment:
we will use the same font for all page content. So no need to define paragraph font family separately. Use `Roboto,"Segoe UI",Helvetica` as the font family for body

Reply:
Okay